src/assets/scripts/client/aircraft/FlightManagementSystem/Fms.js

```diff
@@ -636,6 +636,30 @@ export default class Fms {
         this.currentPhase === FLIGHT_PHASE.CLIMB;
     }
 
+    /**
+     * Facade for #_routeModel.isRunwayModelValidForSid()
```

Comment:
missing bat ticks around `#_routeModel.isRunwayModelValidForSid()`

Reply:
could we also add a blurb about _why_ this facade is needed?  for this and the next one below please.

Reply:
```
Other classes will not have access to the `#_routeModel`, but can use this facade to determine whether or not a given runway assignment is allowable, given the departure procedure in the `#_routeModel`.
```
done